.github/CODEOWNERS

```diff
@@ -22,12 +22,12 @@
 /src/pages/LandingPage* @DataBiosphere/terra-cobranding
 /src/pages/billing/ @DataBiosphere/broadworkspaces-terra-ui
 /src/pages/workspaces/* @DataBiosphere/broadworkspaces-terra-ui
+/src/profile/notification-settings/* @DataBiosphere/broadworkspaces-terra-ui
```

Comment:
Since I have now touched the majority of the code in this directory, and the workspace dashboard depends on the utils methods, adding us a CODEOWNER.